lua/packer/plugin_types/local.lua

```diff
@@ -44,8 +50,17 @@ local function setup_local(plugin)
     end)
   end
 
-  plugin.updater = function(_)
+  plugin.updater = function(disp)
     return async(function()
+      disp:task_update(plugin_name, 'checking symlink...')
+      local resolved_path = vim.fn.resolve(to)
+      vim.pretty_print { to = to, from = from, res = resolved_path }
```

Comment:
Is this supposed to be here or left in by accident?

Reply:
Oh good catch that was there for validation while working on the change.

Reply:
Has been removed and the branch has been rebased